packages/ERTP/src/amountMath.js

```diff
@@ -127,11 +127,10 @@ export const assertValueGetHelpers = value =>
 const optionalBrandCheck = (allegedBrand, brand) => {
   if (brand !== undefined) {
     assertRemotable(brand, 'brand');
-    assert.equal(
-      allegedBrand,
-      brand,
-      X`amount's brand ${allegedBrand} did not match expected brand ${brand}`,
-    );
+    allegedBrand === brand ||
+      Fail`amount's brand ${q(allegedBrand)} did not match expected brand ${q(
```

Comment:
Note change in semantics LOOSENING security: 

The old code was redacting brands. Something as static as a brand I believe does not need to be redacted.